packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1095,5 +1094,97 @@
+	"GO_TO_EDITOR": "Go to editor",
+	"INVALID_EMAIL_ERROR": "{email} is not a valid email",
+	"INVITE": "Invite",
+	"INVITE_NEW_USERS": "Invite new users",
+	"INVITE_RESENT": "Invite resent",
+	"INVITE_USER": "Invite user",
+	"INVITE_X_USER": "Invite {count} user",
+	"LAST_NAME": "Last name",
+	"LOADING": "Loading",
+	"LOGGING_IN_ERROR": "Problem logging in",
+	"MEMBER_ROLE": "Member",
+	"MISSING_INVITE_TOKEN_ERROR": "Missing invite token",
+	"MISSING_TOKEN_ERROR": "Missing token",
+	"MISSING_USERID_ERROR": "Missing user id",
+	"NEW_EMAILS_TO_INVITE": "New user email address(es)",
+	"NEW_PASSWORD": "New password",
+	"NEXT": "Next",
+	"NO_SMTP_TO_SEND_EMAIL_WARNING": "Please contact your admin. n8n isn’t set up to send email right now.",
+	"NO_USERS_TO_INVITE": "No users to invite",
+	"OWNER_ACCOUNT_BENEFITS_MESSAGE": "By setting up an owner account, you can invite other users to join your instance. It also ensures your instance can’t be accessed without an account.",
+	"PASSWORD": "Password",
+	"PASSWORDS_MUST_MATCH_ERROR": "Passwords must match",
+	"DEFAULT_PASSWORD_REQUIREMENTS": "At least 8 characters with 1 number and 1 uppercase",
+	"PASSWORD_UPDATE_ERROR": "Problem changing the password",
+	"PASSWORD_UPDATE_SUCCESS": "Password updated successfully",
+	"PASSWORD_UPDATE_SUCCESS_MESSAGE": "You can now sign in with your new password",
+	"PERSONAL": "Personal",
+	"PERSONAL_SETTINGS": "Personal Settings",
+	"PERSONAL_SETTINGS_UPDATED_SUCCESS": "Updated your details",
+	"PERSONAL_SETTINGS_UPDATE_ERROR": "Problem updating your details",
+	"RECOVERY_EMAIL_SENT": "Recovery email sent",
+	"RECOVER_PASSWORD": "Recover password",
+	"REENTER_NEW_PASSWORD": "Re-enter new password",
+	"RETURN_TO_SIGN_IN": "Back to sign in",
+	"ROLE": "Role",
+	"SAVE": "Save",
+	"SECURITY": "Security",
+	"SENDING_EMAIL_ERROR": "Problem sending email",
+	"SETTINGS": "Settings",
+	"SETTING_UP_OWNER_ERROR": "Problem setting up owner",
+	"SETUP_SMTP_TO_INVITE_USERS_MESSAGE": "Set up SMTP to invite users. <a href=\"https://docs.n8n.io/reference/user-management#smtp\" target=\"_blank\">Instructions</a>",
+	"SET_UP_ACCOUNT_ERROR": "Problem setting up your account",
+	"SET_UP_ACCOUNT_MESSAGE": "Set up your account",
+	"SET_UP_MY_ACCOUNT": "Set up my owner account",
+	"SET_UP_OWNER": "Set up owner account",
+	"SET_UP_TO_INVITE_USERS_WARNING": "Set up an owner account in order to invite other users. Once set up, each user will need to use a password to access this instance.",
+	"SIGN_IN": "Sign in",
+	"SIGN_OUT": "Sign out",
+	"SIGN_OUT_ERROR": "Could not sign out",
+	"SIGN_UP_INVITER_INFO": "{firstName} {lastName} has invited you to n8n",
+	"SKIP_OWNER_SETUP_QUESTION": "Skip owner account setup?",
+	"SKIP_SETUP": "Skip setup",
+	"SKIP_SETUP_TEMPORARILY": "Skip setup for now",
+	"SMTP_TO_ADD_USERS_WARNING": "Set up SMTP before adding users (so that n8n can send them invitation emails). <a target=\"_blank\" href=\"https://docs.n8n.io/reference/user-management#smtp\">Instructions</a>",
```

Elsewhere we refer to them as `invite emails`
```suggestion
	"SMTP_TO_ADD_USERS_WARNING": "Set up SMTP before adding users (so that n8n can send them invite emails). <a target=\"_blank\" href=\"https://docs.n8n.io/reference/user-management#smtp\">Instructions</a>",
```

---

verbatim copied from Max

---

@maxtkacz